### ISSUE TITLE
Tabs won't autosave.  Ubuntu 19.1

### ISSUE THREAD

**Issue by coreycdm** [ReDEnergy/SessionSync#95]
I've tried many times to get autosave to work on my Ubuntu machine but to no avail.  It works just fine on my Windows machine.  Tabs will save if I manually save the session, but other than that, nothing shows in sessions or history.

**Comment by echristopherson** [comment]
That's true here too, using Firefox 83 on Pop!_OS (an Ubuntu variant) 20.04.

**Comment by coreycdm** [comment]
This works for me now.  I wiped/reinstalled to 20.04 as a part of my upgrade cycle.  I'm running FF 84.

**Comment by ReDEnergy** [comment]
It's broken even on Windows, and constantly happens for various users after Firefox update. Reason... well... seems that my code is not the problem and I will have no way to fix it because something is broken in the WebExtension API or it's caused by Firefox itself. Why I say this because the actual issues that prevent me from reading the history tab crashes in the API with a stupid reason

![image](https://user-images.githubusercontent.com/1395510/106121969-d8622f00-6160-11eb-8430-41da1916a06d.png)

If the API's don't work... I have no idea what to do :| . And on top of that, the issues is fixed if you sometimes clear the Firefox profile or reinstall the addon but at that point you lose all the data from history because that's local storage to the addon and once the addon is reinstalled everything is wiped out. Some very smart decision on the API part 🤷‍♂️. And so I'm left with a broken addon that many think it's the fault of the developer Yeyyyy :(

Sorry guys, but can't do anything about this unless there is something caused by me not by the API itself from within Firefox.

**Comment by coreycdm** [comment]
Thanks for the update--I appreciate you explaining the reasoning.  I still love the plugin, even when I was manually saving sessions.  I'm glad that it's back to working automatically after my dev box rebuild!
